src/main/java/com/uber/jenkins/phabricator/conduit/DifferentialClient.java

```diff
@@ -121,4 +142,8 @@ public JSONObject postComment(String revisionID, String message) throws ConduitA
     protected JSONObject callConduit(String methodName, Map<String, String> params) throws ConduitAPIException, IOException {
         return conduit.perform(methodName, params);
     }
+
+    protected JSONObject callConduit(String methodName, JSONObject params) throws ConduitAPIException, IOException {
```

Comment:
Haha agreed.

I think I'm fine with it, I just want to be conscious about the complexity/mental overhead we're introducing where now there are two "obvious" ways to send messages.